cmd.go

```diff
@@ -67,7 +72,7 @@ func processDataCmd(subCmd, context, filename string, progressFunc ProgressCb) (
 		},
 	}
 
-	if subCmd == "get" {
+	if subCmd == "get" || subCmd == "del" {
```

Comment:
That's wrong... To delete a credential you need to put the device into "Memory Management Mode", and deal with the MMM api then. Your path add a del command that does the same as get... What the point?